.secrets.baseline

```diff
@@ -128,6 +132,16 @@
       ]
     }
   ],
-  "results": {},
-  "generated_at": "2024-12-18T09:13:49Z"
+  "results": {
```

Comment:
can't get the json to mark as `pragma: allowlist secret` so updated secret baseline